src/components/molecules/OfferDetails/OfferDetails.tsx

```diff
@@ -126,58 +148,62 @@ export const OfferDetails: FC<IOfferDetails> = ({
             ml: -3,
           }}
         >
-          <picture>
-            <Image
-              sx={{
-                position: 'absolute',
-                top: 0,
-                left: 0,
-                width: '100%',
-                height: '100%',
-                objectFit: 'cover',
-                ...(offerExpired && {
-                  opacity: 0.2,
-                  filter: 'grayscale(1)',
-                }),
-              }}
-              src={imageSlug}
-            />
-          </picture>
-          {offer.company.isGoodCompany && (
-            <Icon
-              name="GoodEmployer"
-              sx={{
-                position: 'absolute',
-                right: ['16px', '24px'],
-                bottom: ['16px', '24px'],
-                width: '145px',
-                height: 'auto',
-                borderRadius: 1,
-                boxShadow: 1,
-                filter: offerExpired ? 'grayscale(1)' : 'initial',
-              }}
-            />
+          {typeof offerImage === 'string' ? (
```

Comment:
jakie mamy wymagania, że offerImage może być string albo JSX?

Reply:
w rekruterze przy dodawaniu oferty nie ma offerImage i wyświetlamy taki placeholder:
https://app.zeplin.io/project/5caf50a54c1f3aa0db033d40/screen/5e7c8f6412a54a8431539472

Reply:
a to placeholder nie może być też po prostu obrazkiem? i wrzucanym do src? albo hostowanym na naszym jakimś serwerze albo nawet po prostu jako plik w waszym projekcie?

Reply:
no własnie po rozmowie z Tomkiem wyszło że lepiej jakby nie był hostowany, dlatego zrobiłam taki placeholder, nie chciałam też dodawać nowego pliku z png do projektu, wolałam zrobić jsx z svg, ale w takim razie do przegadania ;)

Reply:
ok, usunęłam jsx